### PR TITLE
GUI: Remove ANSI color code from TTY

### PR DESCRIPTION
**Pull request by illusion0001**

### CODE REVIEW

**Comment by Megamouse** [comment]
Example?

**Comment by illusion0001** [comment]
Before:
image
After:
image

**Comment by Megamouse** [comment]
idk. the log is there to see the full output of the TTY.
There's no reason to remove tags

**Comment by Megamouse** [comment]
maybe hide it behind a toggle

**Comment by illusion0001** [comment]
Will do, just wanted to make it look slightly prettier in the frame but keep the tags in the log

**Comment by illusion0001** [comment]
> maybe hide it behind a toggle

added

File: rpcs3/rpcs3qt/log_frame.cpp [hunk 1]
```diff
@@ -544,7 +555,18 @@ void log_frame::UpdateUI()
 				// clear selection or else it will get colorized as well
 				text_cursor.clearSelection();
 
-				const QString tty_text = QString::fromStdString(buf_line);
+				// Strip ANSI color code
+				QRegularExpression ansi_color_code("\\[\\d+;\\d+(;\\d+)?m|\\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]");
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/log_frame.cpp:559]: please put this into the correct scope.
Also, it can be static const or even constexpr

Comment (illusion0001) [reply rpcs3/rpcs3qt/log_frame.cpp]: fixed

**Pull request merged**